commitlintplugins.test.ts

```diff
@@ -152,6 +152,28 @@ test('prefer-slash-over-backslash2', () => {
 });
 
 
+test('reject-hashtag-refs1', () => {
+    let commitMsgWithHashtagRef = "foo: fixes #123";
```

Comment:
@realmarv woah, no, let's not have a test that has it in the commit title, because we cannot suggest URLs to be added to the commit title (they would make the title too long for sure)